projectOne.js

```diff
@@ -1,42 +0,0 @@
-const readline = require("readline");
-const r1 = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-   return new Promise ((resolve, reject) => {
-      r1.question(questionText, resolve);
-  });
-}
-console.log ('Lets play a game \n You will think of a number between 1 and 100. \nI will guess your numbre in 7 tries or less./n My abilitis are very, very, very good i.e. excellent')
-
-start();
- 
-async function start() {
-   let secretNumber = await ask('Do you have your secret number?\nDon\'t worry, I won\'t peek...\n type in Y if Yes or N if you do not') 
-
-   console.log ('[Yes, I have a number"')
-   let max=100
-   let min= 1
-   let guess = Math.floor((max-min)/2) // An efficent guess
```

Comment:
This is really close to being the proper math for a smart guess, You will need to change it slightly though to be more expandable and more accurate. we will want it to be `Math.floor((max+min)/2)`. Doing this will allow you to have numbers other than 100 and 1 as your max and mid and still give you the most accurate center point